### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by AykutSarac** [AykutSarac/jsoncrack.com#244] [Announcements]
## JSON Crack - v2.1.0
* Dynamic widgets! (Older embeds are now **deprecated**!) Explore more: https://jsoncrack.com/embed
* Fix expand/collapse parents
* Add collapse all parents to sidebar
* Mobile support
* Performance improvements
* JSONC support (JSON with comments)
* Display children count on parent
* Fix Center View

### [**❤️ SUPPORT JSON CRACK!**](https://github.com/sponsors/AykutSarac)

<img width="1728" alt="Screen Shot 2022-10-26 at 17 18 28" src="https://user-images.githubusercontent.com/47941171/198079843-b7406d81-e9be-46d7-8686-68f6700db3d5.png">

## What's Changed
* fix hide wrong collapsed nodes by @victorbrambati in https://github.com/AykutSarac/jsoncrack.com/pull/107
* jsoncrack.com Migration by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/113
* Add mobile support for JSON Crack by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/115
* Enable Performance Mode by default by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/132
* General UI/UX Improvements by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/133
* Link Travis CI badge to project build status by @mbrukman in https://github.com/AykutSarac/jsoncrack.com/pull/109
* Update Docker section formatting in README by @mbrukman in https://github.com/AykutSarac/jsoncrack.com/pull/108
* Bug/share link by @yuvalg99 in https://github.com/AykutSarac/jsoncrack.com/pull/136
* Improve Editor Rerender Performance by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/157
* Prevent All Nodes from Expanding by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/167
* fix: highlight red-dashed border display by @Hexrt in https://github.com/AykutSarac/jsoncrack.com/pull/165
* Handle window freeze by @AykutSarac in https://github.com/AykutSarac/jsoncrack.com/pull/191
* Canvas should fit into view on click Center icon at toolbar by @dogukanuhn in https://github.com/AykutSarac/jsoncrack.com/pull/203
* Accept JSONC by stripping it of it's comments by @VladCuciureanu in https://github.com/AykutSarac/jsoncrack.com/pull/227
* Added children count label next to the node key by @VladCuciureanu in https://github.com/AykutSarac/jsoncrack.com/pull/218
* Added a 'Collapse/Expand Graph' button by @VladCuciureanu in https://github.com/AykutSarac/jsoncrack.com/pull/230
* Fix: persist hidden nodes on rotate by @shivam1646 in https://github.com/AykutSarac/jsoncrack.com/pull/238
* Fix: don't show disconnected nodes on expand by @shivam1646 in https://github.com/AykutSarac/jsoncrack.com/pull/243

## New Contributors
* @mbrukman made their first contribution in https://github.com/AykutSarac/jsoncrack.com/pull/109
* @yuvalg99 made their first contribution in https://github.com/AykutSarac/jsoncrack.com/pull/136
* @Hexrt made their first contribution in https://github.com/AykutSarac/jsoncrack.com/pull/165
* @dogukanuhn made their first contribution in https://github.com/AykutSarac/jsoncrack.com/pull/203
* @VladCuciureanu made their first contribution in https://github.com/AykutSarac/jsoncrack.com/pull/227
* @shivam1646 made their first contribution in https://github.com/AykutSarac/jsoncrack.com/pull/238

**Full Changelog**: https://github.com/AykutSarac/jsoncrack.com/compare/v2.0.0...v2.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AykutSarac/jsoncrack.com/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by SwanseN21** [comment]
Fantastic
